### DISCUSSION TITLE
Difference between the newly added "animations" and "scroll".

### DISCUSSION THREAD

**Discussion by redoxahmii** [LazyVim/LazyVim#5066] [Q&A] [answered]
I can't seem to distinguish between what the Animations Toggle provide other than Smooth Scroll.
And if there is any more things that are being animated how can we disable individually because Smooth Scroll is enabled by default which I don't think is a good default and if i disable it using the config for `snacks.nvim` by passing it :
```lua
scroll = {
        enabled = false,
      }
```
Is this disabling the other animations as well?
I am not sure if there are any other atm.
There is also the method provided in options but that disables every animation as well I just wish it could remember the last state of the Smooth Scroll toggle or just have that disabled by default as it just slows you down if you're scrolling through a buffer and is still a bit janky being a terminal at the end where it is running but that's my opinion.

**Comment by folke** [comment, marked as answer]
To disable all animations, you can set `vim.g.snacks_animate = false` in your `options.lua`.
If you want to disable just scroll, you can do what you have.
To disable animations separately for the other plugins, check the snacks `dim` and snakcs `indent` docs.
